apps/wing/src/commands/run.ts

```diff
@@ -49,19 +49,27 @@ export async function run(entrypoint?: string, options?: RunOptions) {
   entrypoint = resolve(entrypoint);
   debug("opening the wing console with:" + entrypoint);
 
-  const { port } = await createConsoleApp({
+  const { port, close } = await createConsoleApp({
     wingfile: entrypoint,
     requestedPort,
     hostUtils: {
-      async openExternal(url) {
+      async openExternal(url: string) {
         await open(url);
       },
     },
-    requireAcceptTerms: true,
+    requireAcceptTerms: !!process.stdin.isTTY,
```

Comment:
Is this for unit testing? or just an unrelated bug we had?

Reply:
No related bug for it afaik, and it's not for testing. I made the change in search of root cause, and left it in cause it seemed like a better thing to do.